mail_show_follower/models/mail_mail.py

```diff
@@ -104,8 +104,10 @@ def _send(self, auto_commit=False, raise_exception=False, smtp_session=None):
                         # get partners
                         cc_internal = True
                         # else get company in object
+                        company = self.env.company
```

Comment:
I think you can refactor this code to get the put the logic to get the company on one side, and then access to the properties then, so it's clearer. If not, all this `if` are hard to read and prone to errors.